vetomint/src/lib.rs

```diff
@@ -33,21 +33,17 @@ pub enum ConsensusEvent {
     /// Informs that the node has received a block proposal.
     BlockProposalReceived {
         proposal: BlockIdentifier,
+        /// Whether this proposal was valid or locked in this round.
+        proposal_round: Option<Round>,
         proposer: ValidatorIndex,
         round: Round,
         time: Timestamp,
-    },
-    /// Informs that the node is in favor of or against a proposal.
-    ProposalFavor {
-        proposal: BlockIdentifier,
         /// Whether this node is in favor of the proposal.
         favor: bool,
-        time: Timestamp,
     },
-    /// Informs that `CreateProposal` has been completed.
-    BlockProposalCreated {
+    /// Updates the block candidate which this node wants to propose in its turn.
```

Comment:
Updates the block candidate in which this nodes wants to propose

Reply:
fix it